Dockerfile

```diff
@@ -8,14 +10,17 @@ COPY package.json package-lock.json /usr/src/app/
 # Without the badge-maker package.json and CLI script in place, `npm ci` will fail.
 COPY badge-maker /usr/src/app/badge-maker/
 
-RUN npm install -g "npm@^10"
+COPY . /usr/src/app
+
 # We need dev deps to build the front end. We don't need Cypress, though.
-RUN NODE_ENV=development CYPRESS_INSTALL_BINARY=0 npm ci
+RUN NODE_ENV=development CYPRESS_INSTALL_BINARY=0 npm ci \
+    && npm run build \
+    && npm prune --omit=dev \
```

Comment:
If we use later `rm -rf node_modules` there is no point in using prune or clearing cache here.

Reply:
altho we might do use it, see next comment about how we might be able to not remove node_module completely.